src/components/UI/Layout/Modal/index.tsx

```diff
@@ -15,6 +15,22 @@ const CloseButtonText = styled(Text)`
   line-height: normal;
 `;
 
+const Content = styled(Box)`
+  ::-webkit-scrollbar {
```

Comment:
*cries in firefox*

Reply:
I added some properties to support Firefox as well, but I don't think there's an equivalent property to style scrollbar radius. It looked a bit obtrusive with the sharp corners, so I made the scrollbar track transparent instead:
<img width="400" alt="Screen Shot 2021-11-09 at 11 59 29" src="https://user-images.githubusercontent.com/62935115/140913550-6709f508-a9f2-408f-92b3-6e82e08ebff2.png">